processor/runtime/src/main/java/com/redhat/service/smartevents/processor/actions/webhook/WebhookActionInvoker.java

```diff
@@ -47,7 +57,23 @@ public void onEvent(String event) {
         } else if (basicAuthUsername != null) {
             request.basicAuthentication(basicAuthUsername, basicAuthPassword);
         }
-        request.sendJsonObjectAndForget(new JsonObject(event));
+
+        // Add our HTTP Headers.
+        // This can be replaced with w3c trace-context parameters when we add distributed tracing.
+        request.headers().add(APIConstants.X_RHOSE_BRIDGE_ID, bridgeId);
```

Comment:
This probably isn't needed for HTTP'ing... but lets add it for consistency (with the Kafka invoker).